addons/website_slides/static/src/js/slides_embed.js

```diff
@@ -226,8 +226,15 @@ $(function () {
         $('.oe_slide_js_share_email button').on('click', function () {
             var widget = $('.oe_slide_js_share_email');
             var input = widget.find('input');
+            const inputValues = input.val().split(",");
+            let areEmailsValid = true;
             var slideID = widget.find('button').data('slide-id');
-            if (input.val() && input[0].checkValidity()) {
+            inputValues.forEach(email => {
+                if (!email.match(/.+@.+/)) {
+                    areEmailsValid = false;
+                }
+            });
+            if (input.val() && areEmailsValid) {
```

Comment:
This code is copy / paste 3 times in this commit, isn't it better to do the verification in `/slides/slide/send_share_email` and return an error if one of the email is not valid ?

With this you can also use the existing email regex in `tool/mail.py`, again we will remove redundancy :)

What do you think ?